components/tidb_query_datatype/src/codec/data_type/vector.rs

```diff
@@ -260,8 +260,22 @@ impl VectorValue {
                 }
                 size
             }
-            // TODO: implement here after we implement enum/set encoding
-            VectorValue::Enum(_) => unimplemented!(),
+            VectorValue::Enum(vec) => {
+                let mut size = logical_rows.len() + 10;
```

Comment:
Sorry I forgot to update this part😂.
I think maybe `logical_rows.len() * 9 + logical_rows.len() * 8 + sum(string_length) + 10` instead? Since we write both int and string to chunks.